packages/zosjobs/src/SubmitJobs.ts

```diff
@@ -119,6 +124,12 @@ export class SubmitJobs {
             // default to fixed format records
             headers.push(ZosmfHeaders.X_IBM_INTRDR_RECFM_F);
         }
+        if (parms.jclSymbols) {
+            const extraHeaders = this.getSubstitutionHeaders(parms.jclSymbols);
+            for (const header of extraHeaders) {
+                headers.push(header);
+            }
```

Comment:
This could be replaced with one line: `headers.push(...extraHeaders);`

No need to change, just pointing out an ES6 trick 😛

Reply:
Neat, didn't know that it'd work that way with the spread operator.

Reply:
This was easy enough to change while doing other things.